*** ./smali/android/app/SystemServiceRegistry.smali	2017-02-15 19:18:30.831446258 +0530
--- /home/ram/MIUI8_patchROM/Note4/temp/new_smali/framework.jar.out/./smali/android/app/SystemServiceRegistry.smali	2017-02-15 19:25:06.255236435 +0530
***************
*** 750,755 ****
--- 750,757 ----
  
      invoke-static {v0, v1, v2}, Landroid/app/SystemServiceRegistry;->registerService(Ljava/lang/String;Ljava/lang/Class;Landroid/app/SystemServiceRegistry$ServiceFetcher;)V
  
+     invoke-static {}, Landroid/app/ContextImplInjector;->registerMiuiServices()V
+ 
      return-void
  .end method
  
